.github/workflows/release-perform.yml

```diff
@@ -0,0 +1,105 @@
+name: Quarkiverse Perform Release
+run-name: Perform ${{github.event.inputs.tag || github.ref_name}} Release
+on:
+  push:
+    tags:
+      - '*'
+  workflow_dispatch:
+    inputs:
+      tag:
+        description: 'Tag to release'
+        required: true
+
+permissions:
+  attestations: write
+  id-token: write
+  contents: read
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  perform-release:
+    name: Perform Release
+    uses: quarkiverse/.github/.github/workflows/perform-release.yml@main
+    secrets: inherit
+    with:
+      version: ${{github.event.inputs.tag || github.ref_name}}
+
+  prepare-platform-pr:
+    name: Prepare Platform Update
+    runs-on: ubuntu-latest
+
+    outputs:
+      released: ${{steps.wait-for-bom.outputs.published}}
+      qosdk-version: ${{steps.metadata.outputs.current-version}}
+      quarkus-platform-branch: ${{steps.versions.outputs.quarkus_platform_branch}}
+      needs-main-pr: ${{ steps.versions.outputs.needs_main_pr == 'true' }}
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - uses: radcortez/project-metadata-action@main
+        name: Retrieve project metadata
+        id: metadata
+        with:
+          github-token: ${{secrets.GITHUB_TOKEN}}
+          metadata-file-path: '.github/project.yml'
+          local-file: true
+
+      - name: Gather versions
+        id: versions
+        run: |
+          echo "quarkus_version=$(./mvnw help:evaluate -Dexpression=quarkus.version -q -DforceStdout)" >> $GITHUB_OUTPUT
```

Comment:
You can save some time by specifying `${{github.event.inputs.tag || github.ref_name}}` directly here

Reply:
I don't think this would work here as this specific line is supposed to retrieve the Quarkus version defined in the POM, not the QOSDK version.

Reply:
Ah you're right, I missed that. Good call